src/SFA.DAS.Reservations.Domain/Reservations/Reservation.cs

```diff
@@ -14,5 +15,36 @@ public class Reservation
     {
         public Guid Id { get; set; }
         public ReservationStatus Status { get; set; }
+        public long AccountId { get; set; }
+        public long AccountLegalEntityId { get; set; }
+        public string AccountLegalEntityName { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public DateTime CreatedDate { get; set; }
+        public string CourseId { get; set; }
+        public string CourseName { get; set; }
+        public int? CourseLevel { get; set; }
+        public uint? ProviderId { get; set; }
+        public bool EmployerDeleted { get; set; }
+
+        public static implicit operator Reservation(ReservationCreatedEvent source)
+        {
+            return new Reservation            
+            {
+                Id = source.Id,
+                Status = ReservationStatus.Pending,
+                AccountId = source.AccountId,
+                AccountLegalEntityId = source.AccountLegalEntityId,
+                AccountLegalEntityName = source.AccountLegalEntityName,
+                StartDate = source.StartDate,
+                EndDate = source.EndDate,
+                CreatedDate = source.CreatedDate,
+                CourseId = source.CourseId,
+                CourseName = source.CourseName,
+                CourseLevel = int.Parse(source.CourseLevel),
```

Comment:
Given course level could be null in the event should we not be handling null values?